src/main/asciidoc/repository-query-return-types-reference.adoc

```diff
@@ -14,20 +14,19 @@ NOTE: Geospatial types like (`GeoResult`, `GeoResults`, `GeoPage`) are only avai
 |`void`|Denotes no return value.
 |Primitives|Java primitives.
 |Wrapper types|Java wrapper types.
-|`T`|An unique entity. Expects the query method to return one result at most. In case no result is found `null` is returned. More than one result will trigger an `IncorrectResultSizeDataAccessException`.
+|`T`|An unique entity. Expects the query method to return one result at most. If no result is found, `null` is returned. More than one result triggers an `IncorrectResultSizeDataAccessException`.
 |`Iterator<T>`|An `Iterator`.
 |`Collection<T>`|A `Collection`.
 |`List<T>`|A `List`.
-|`Optional<T>`|A Java 8 or Guava `Optional`. Expects the query method to return one result at most. In case no result is found `Optional.empty()`/`Optional.absent()` is returned. More than one result will trigger an `IncorrectResultSizeDataAccessException`.
-|`Option<T>`|An either Scala or JavaSlang `Option` type. Semantically same behavior as Java 8's `Optional` described above.
+|`Optional<T>`|A Java 8 or Guava `Optional`. Expects the query method to return one result at most. If no result is found, `Optional.empty()` or `Optional.absent()` is returned. More than one result triggers an `IncorrectResultSizeDataAccessException`.
+|`Option<T>`|Either a Scala or JavaSlang `Option` type. Semantically the same behavior as Java 8's `Optional`, described earlier.
```

Comment:
Nit: Align JavaSlang to Javaslang (as mentioned in an earlier chapter)

Reply:
Yup. Thanks. (I checked, and it's Javaslang).